UKHomeOffice/ide-controller
# COP-1977: Add clear button

## Description
Add the "Clear Data Now" button to the UI.
## To Test
- Install app and run dev (this will open an electron app window)
- If this page is blank, refresh the page with cmd + r (mac) or ctl + r (windows)
- You should see the "warning" button beneath the "further information" table on the left

## Developer Checklist

\* Required
